src/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSource.csproj

```diff
@@ -5,6 +5,11 @@
     <UseOpenKey>true</UseOpenKey>
     <ProjectGuid>{3DF9A5D5-3D4B-4378-9B55-CFA6AC0114D9}</ProjectGuid>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetGroup)' == 'net45' OR '$(TargetGroup)' == 'net46' OR '$(TargetGroup)' == 'netfx'">
+     <DefineConstants>;ALLOW_PARTIALLY_TRUSTED_CALLERS</DefineConstants>
```

Comment:
You should change this to `<DefineConstants>$(DefineConstants);ALLOW_PARTIALLY_TRUSTED_CALLERS</DefineConstants>` otherwise you will drop any other constants defined elsewhere.